tsdb/block.go

```diff
@@ -419,6 +422,11 @@ func (r blockIndexReader) Symbols() index.StringIter {
 	return r.ir.Symbols()
 }
 
+func (r blockIndexReader) SortedLabelValues(name string) ([]string, error) {
+	st, err := r.ir.SortedLabelValues(name)
+	return st, errors.Wrapf(err, "block: %s", r.b.Meta().ULID)
```

Comment:
We are wrapping potentially nil, do you mind adding 3 lines more? :thinking:

Reply:
Wrapping nil is fine and returns nil. It's a common usage of this function.

Reply:
It returns nil for nil errors, and this is not a highly called method like the sample appending where this can cause an overhead. So I think this should be fine.

Reply:
I know this works, just it's something I would vote to clear for readability: See https://github.com/thanos-io/thanos/pull/2630

not a blocker.